pkg/ebpf/c/tracee.bpf.c

```diff
@@ -3960,6 +3964,21 @@ int BPF_KPROBE(trace_ret_vfs_writev_tail)
     return do_vfs_write_writev_tail(ctx, VFS_WRITEV);
 }
 
+SEC("kprobe/__kernel_write")
+TRACE_ENT_FUNC(kernel_write, KERNEL_WRITE);
+
+SEC("kretprobe/__kernel_write")
+int BPF_KPROBE(trace_ret_kernel_write)
+{
+    return do_vfs_write_writev(ctx, KERNEL_WRITE, TAIL_KERNEL_WRITE);
```

Comment:
the only difference i see between the `__kernel_write` function and the `vfs_write` one, is that the `buf` argument is a user pointer in vfs_write. this shouldn't cause a problem because of the use of `bpf_probe_read` in `do_vfs_write_writev` func.

i guess maybe just renaming `do_vfs_write_writev` to something more general would make it clearer.